tests/templates/subroutines/qchem/test_all_singles_doubles.py

```diff
@@ -191,6 +192,24 @@ def circuit2():
 class TestInputs:
     """Test inputs and pre-processing."""
 
+    @pytest.mark.jax
```

Comment:
Why do we have `jax` marker here?

Reply:
Need jax to do the full `assert_valid` checks. Honestly, a lot of problems have been hidden because we weren't running `assert_valid` with jax installed 😢